### PR TITLE
Allow both top and bottom positioning for banner

### PR DESCRIPTION
**Pull request by intrepidws**
This PR allows the user to specify the position (either `top` or `bottom`) for the banner. This will continue to default to the top.

### CODE REVIEW

**Comment by jszobody** [comment]
Nice! One issue I see is that you probably need to update this as well.

https://github.com/stechstudio/filament-impersonate/blob/f4052c3f27c3351e95003a1f3fdc025255bf627e/resources/views/components/banner.blade.php#L13

I'm making room for the top fixed banner by pushing down the rest of the page. If you move that to the bottom, you at least want to remove this 50px top space. Not sure if it makes sense to add space at the bottom or not, you can test that.

File: resources/views/components/banner.blade.php [hunk 1]
```diff
@@ -10,8 +10,8 @@
 @endphp
 
 <style>
-    html {
-        margin-top: 50px;
+    aside, body {
```
Comment (jszobody) [inline resources/views/components/banner.blade.php:13]: Interesting, why include `aside` here?

**Comment by intrepidws** [comment]
@jszobody Good call, thanks for the input. I noticed that the sidebar wasn't actually being pushed down with that margin, so I changed how it worked and also accounted for how it should react when the banner is positioned at the bottom.

File: resources/views/components/banner.blade.php (hunk 1 above)
Comment (intrepidws) [reply resources/views/components/banner.blade.php:13]: The sidebar wasn't actually being pushed down with how it was before.

**Comment by jszobody** [comment]
Ahhh ok. So one point of clarification then.

The `banner` component is used both by Filament and other UIs. For example, I'm injecting that in a client-facing, non-Filament UI. I don't think you want to be globally styling `aside` there.

Instead, you want to add any Filament-specific styling here:

https://github.com/stechstudio/filament-impersonate/blob/master/resources/views/auto-inject-banner.blade.php

That's the one we auto-inject into Filament specifically, and can add styling that doesn't then get applied in non-Filament environments.

**Comment by intrepidws** [comment]
OK so I took a bit of a different approach here. I wasn't seeing the Filament specific styles because my path for Filament is set to empty, so it serves from the root domain. Rather than rely on the Filament path config variable, I thought that we could simply make the CSS rules more specific to reference the Filament classes. Thoughts?

**Comment by jszobody** [comment]
That makes a lot of sense. I hadn't actually noticed the filament classes when I wrote this. 🤦

**Pull request merged**